bson/src/main/org/bson/BsonReader.java

```diff
@@ -19,12 +19,14 @@
 import org.bson.types.Decimal128;
 import org.bson.types.ObjectId;
 
+import java.io.Closeable;
+
 /**
  * An interface for reading a logical BSON document using a pull-oriented API.
  *
  * @since 3.0
  */
-public interface BsonReader {
+public interface BsonReader extends Closeable {
```

Comment:
Backwards breaking for anyone implementing BsonReader from scratch, doubtful anyone is.